project.html

```diff
@@ -0,0 +1,512 @@
+                <img class="best-destinations-photo photo" src="images/destinations_images/Island.svg" alt="Island">
+                </a>
+                <p class="home-hotel-name home-text"><a href="#">Island</a></p>
+            </div>
+
+            <div class="home-switcher">
+                <a href="#">
+            <img class="home-switcher-dest" src="images/homes_images/arrow.svg" alt="arrow" width="40" height="40">
+                </a>
+            </div>
+    </div>
+
+</section>
+
+<div class="sign-up-reviews">
+
+<section class="sign-up">
+
+    <div class="sign-up-container">
+        <img class="sign-up-closer closer" src="images/sign-up_images/closer.svg" alt="closer" width="26" height="26">
+        <p class="sign-up-text">Sign up and get 20% off your first booking</p>
+        <a href="#">
+        <img class="sign-up-icon" src="images/sign-up_images/user.svg" alt="icon">
+        </a>
+        <form>
+            <button class="sign-up-btn">Sign up</button>
+        </form>
+    </div>
+
+    <div class="sign-up-container-adapt">
+        <img class="sign-up-closer-adapt closer" src="images/sign-up_images/closer.svg" alt="closer" width="16" height="16">
+        <a href="#">
+        <img class="sign-up-logo-adapt" src="images/sign-up_images/logo_small.svg" alt="logo" width="60" height="60">
+        </a>
+        <p class="sign-up-text-adapt">Sign up in the app and get 20% <br>off your first booking</p>
+        <form>
+            <button class="sign-up-install-adapt">Install</button>
+        </form>
+    </div>
+
+</section>
+
+<section class="reviews">
+    <svg class="svg" width="53" height="53"  fill="none" xmlns="http://www.w3.org/2000/svg" style="position:absolute; visibility:hidden; display:none; width:0; height:0;">
+        <symbol id="reviews-account" viewBox="0 0 53 53">
+            <path d="M 27.0002 0.333252 C 12.2802 0.333252 0.333496 12.2799 0.333496 26.9999 C 0.333496 41.7199 12.2802 53.6666 27.0002 53.6666 C 41.7202 53.6666 53.6668 41.7199 53.6668 26.9999 C 53.6668 12.2799 41.7202 0.333252 27.0002 0.333252 Z M 27.0002 8.33325 C 31.4268 8.33325 35.0002 11.9066 35.0002 16.3333 C 35.0002 20.7599 31.4268 24.3333 27.0002 24.3333 C 22.5735 24.3333 19.0002 20.7599 19.0002 16.3333 C 19.0002 11.9066 22.5735 8.33325 27.0002 8.33325 Z M 27.0002 46.1999 C 20.3335 46.1999 14.4402 42.7866 11.0002 37.6133 C 11.0802 32.3066 21.6668 29.3999 27.0002 29.3999 C 32.3068 29.3999 42.9202 32.3066 43.0002 37.6133 C 39.5602 42.7866 33.6668 46.1999 27.0002 46.1999 Z"></path>
+        </symbol>
+    </svg>
+    <div class="reviews-container">
+    <h1 class="reviews-title">Guest reviews</h1>
+    <h1 class="reviews-title-adapt">Reviews</h1>
+    <div class="reviews-gallery">
+        <div class="reviews-box">
+            <a href="#">
+            <div class="reviews-cost">From 250 USD</div>
+            <img class="reviews-image" src="images/reviews_images/Bali.svg" alt="Bali" width="400" height="400">
```

Comment:
If I load a random photo (which real users will obviosly do), it doesn't make my photo square and UI looks broken 
![image](https://user-images.githubusercontent.com/36200303/113488467-0b110680-94c7-11eb-831f-5a54dee7f3ef.png)